scripts/epa/airdata/criteria_gases/import_config.txt

```diff
@@ -0,0 +1,15 @@
+import: {
+  import_name: "EPACriteriaGases3"
+  provenance_url: "https://aqs.epa.gov"
```

Comment:
nit: maybe sync with the comments from google3 manifest CL

Reply:
done